git/shas_from_filepath.go

```diff
@@ -0,0 +1,34 @@
+package git
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+)
+
+// GetShasFromFilepath opens the file at path and reads all the git repos and SHAs in the file. Returns a slice of all the RepoAndShas that correspond to each entry in the file, or an empty slice and non-nil error if there was any error
```

Comment:
@mboersma fixed in https://github.com/deis/deisrel/pull/89/commits/79bf288c9d95b2fb31f7bfd40ec33cc9f2b7eaed